python/tank/platform/engine.py

```diff
@@ -1219,6 +1212,53 @@ def _do_invoke(self):
         # don't have ui so can't create an invoker!
         return None
 
+    def _generate_panel_id(self, app_instance_name, panel_name):
+        """
+        Compose a unique id for a panel. This is done based on the app instance
+        name plus the given panel name. By using the instance name rather than
+        the app name, we support the use case where more than one instance of an
+        app exists within a config.
+        :param app_instance_name: Instance name of the current application.
```

Comment:
please add a blank line between the description and `:param:` section for readability